### DISCUSSION TITLE
Cant see task instance details page

### DISCUSSION THREAD

**Discussion by imranshaikmuma** [apache/airflow#21133] [General]
### Apache Airflow version

2.0.2

### What happened

Cant see task instance details page
The retry of the task failed because the task instance is in failed status. But we cant see the log of task instance as the page is not being rendered. Cloudwatch log has no logs as well

### Operating System

Amazon Linux AMI

### Deployment

MWAA

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
There is no reproducibility, so it is at most discussion. I think you will need to provide more details if you need help as there is no way anyone could act on the description to help you.
